solc/CommandLineParser.cpp

```diff
@@ -1460,6 +1489,61 @@ void CommandLineParser::processArgs()
 		m_options.input.mode == InputMode::CompilerWithASTImport ||
 		m_options.input.mode == InputMode::EVMAssemblerJSON
 	);
+
+	std::string ethdebugOutputSelection =
```

Comment:
this is only used in error messages so i would restrict its scope to that or at least make it static and reflect that it's a message in the variable's name